base.mk

```diff
@@ -23,8 +23,7 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.codename=$(PLATFORM_VERSION_CODENAME) \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
-	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
```

Comment:
that's really not what I meant...
I wanted you to **edit** your **commit** to remove the **change** you had there.
I don't want to remove ro.adb.secure, it's fine there.
But you had a change with various whitespaces
I don't want to squash your commits, I want them individual, hence I want you to really edit commits (and thus remove this commit)